src/RemoteSsh.cpp

```diff
@@ -131,13 +131,13 @@ void SshConnection::connect()
         args.append(SCAP_WORKBENCH_LOCAL_SSH_PATH);
 #endif
 
-        args.append("-M"); // place ssh client into "master" mode for connection sharing
+        args.append("-M"); // place ssh client into "leader" mode for connection sharing
```

Comment:
I found no evidence of a Leader mode in SSH.

```suggestion
        args.append("-M"); // place ssh client in control of the shared connection socket
```

Reply:
will update it accordingly